routes/chorals.js

```diff
@@ -323,8 +323,18 @@ router.delete('/:choralId', function(req, res, next) {
         return res.send('500'); // notify client of failure
       }
 
-      res.flash('success', 'Choral successfully deleted.');
-      return res.send('204');  // successful deletion
+      // Delete all children choral from parents
```

Comment:
I think it would make more sense to say "Delete this child choral from any parents it has"